filters/scheduler/fifo.go

```diff
@@ -151,3 +151,8 @@ func (f *fifoFilter) Response(ctx filters.FilterContext) {
 	pending[last]()
 	ctx.StateBag()[fifoKey] = pending[:last]
 }
+
+// HandleErrorResponse is to opt-in for filters to get called
+// Response(ctx) in case of errors via proxy. It has to return true to
+// opt-in.
+func (f *fifoFilter) HandleErrorResponse() bool { return true }
```

Comment:
Where this function is called from?

Reply:
https://github.com/zalando/skipper/pull/2239

Reply:
proxy/proxy.go:1580:            if ehf, ok := fi.Filter.(errorHandlerFilter); !ok || !ehf.HandleErrorResponse() {